js/cd-print.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @file
+ * Print helper.
+ */
+
+((Drupal) => {
+  'use strict';
+
+  Drupal.behaviors.cdPrint = {
+    attach: function (context, settings) {
+
+      // Remove lazy loading before printing.
+      window.addEventListener('beforeprint', (event) => {
+        $('img').each(() => {
+          $(this).removeAttr('loading');
```

Comment:
`$` assumes that jquery is globally available. Iframe also can be lazy [loaded](https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement/loading).

So better be more generic and use vanilla JS:

```suggestion
        document.querySelectorAll('[loading="lazy"]').forEach(element => {
          element.removeAttribute('loading');
        });
```

**Notes**

This may not be enough to force load the images and iframes, depending on the browsers. This needs testing and if that doesn't work, re-setting the `src` attribute for example may do the trick.

This may also require adjusting the waiting time on the snap tool.

Reply:
The patch is now on the feature instance, where it seems to be working as intended: https://stage.response-reliefweb-int.ahconu.org/fr/central-african-republic/bulletin-humanitaire/reponse-durgence